### PR TITLE
Fix grammar in the comments in TIA-MC1 and Instrebiteli drivers

### PR DESCRIPTION
**Pull request by sev-**
I went through the comments for TIA-MC1 and Istrebiteli drivers and fixed their grammar.

No functional changes were brought.

### CODE REVIEW

**Comment by cuavas** [comment]
There are more spelling and grammar errors introduced in this PR, and changing C++ line comments to C style isn't helpful.  You've also changed the meaning of several comments in addition to changing grammar.

**Comment by sev-** [comment]
Weird reaction. I am the original author of tiamc1 file, and yes, I wanted uniformity, as the latter additions introduced mix of the comment style.

Could you please clarify, where I changed meaning and where I introduced more spelling errors?

File: src/mame/drivers/tiamc1.cpp [hunk 1]
```diff
@@ -486,35 +486,35 @@ ROM_START( gorodki )
...
 	ROM_LOAD( "prom101.a01", 0x0100, 0x100, NO_DUMP ) /* video sync 256x8 */
-	ROM_LOAD( "prom102.b03", 0x0200, 0x080, NO_DUMP ) /* sprites rom index 256x4 */
+	ROM_LOAD( "prom102.b03", 0x0200, 0x080, NO_DUMP ) /* sprite rom index 256x4 */
 	ROM_LOAD( "prom102.b06", 0x0280, 0x080, NO_DUMP ) /* buffer optimization logic 256x4 */
-	ROM_LOAD( "prom102.b05", 0x0300, 0x100, NO_DUMP ) /* sprites rom index 256x8 */
+	ROM_LOAD( "prom102.b05", 0x0300, 0x100, NO_DUMP ) /* sprite rom index 256x8 */
 ROM_END
 
 /*
-  later (cost reduced ?) version of TIA-MC1 hardware, uses single PCB main board design,
```
Comment (cuavas) [inline src/mame/drivers/tiamc1.cpp]: "Later" is correct here

File: src/mame/drivers/tiamc1.cpp [hunk 2]
```diff
@@ -331,7 +331,7 @@ static MACHINE_CONFIG_START( tiamc1, tiamc1_state )
 
 	/* video hardware */
 	MCFG_SCREEN_ADD("screen", RASTER)
-	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       // pixel clock and htotal comes from docs/schematics, the rest is guess (determined by undumped PROM)
+	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       /* pixel clock and htotal come from docs/schematics, the rest is a guesswork (originall specified by an undumped PROM) */
```
Comment (cuavas) [inline src/mame/drivers/tiamc1.cpp]: It should be "the rest is a guess" or "the rest is guesswork".  "the rest is a guesswork" is bad grammar.  Also, "originall" is a typo.

File: src/mame/drivers/istrebiteli.cpp [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
+      hit enemy 15 or more times to get the bonus game
 
     test mode:
       insert 12 or more coins then press 2 player start
 
     notes:
-      dumped PCB is early game version, have several bugs, possible test/prototype.
-      later version was seen in St.Petersburg arcade museum, CPU board have single 8Kx8 ROM.
+      dumped PCB is from an early game version which has several bugs, perhaps it is a test/prototype.
+      latter version was seen in the St.Petersburg arcade museum, CPU board has a single 8Kx8 ROM.
```
Comment (cuavas) [inline src/mame/drivers/istrebiteli.cpp]: "later" is correct here

**Comment by cuavas** [comment]
I've pointed out introduced errors with comments against the commit.

C++ line comments are better for notes at the end of lines of code because you can easily block-comment them out, and they're less visual clutter as they don't have an extra visual token at the end of the text.

My biggest gripe though was the introduction of grammatical and spelling errors by a PR that claimed to address the same.

**Pull request closed without merging**